### PR TITLE
react_decompose

### PR DESCRIPTION
**Pull request by dima-korovai**
[DEMO LINK](https://dima-korovai.github.io/react_decompose/)

### CODE REVIEW

**Review by polosanya — CHANGES_REQUESTED** [review]
Please add a demo link to your PR description so that we can check your work properly)
Also, let's fix this problem to pass the tests
image

File: src/components/Article/Article.css [hunk 1]
```diff
@@ -1 +1,31 @@
...
+    padding: 2em;
+  }
+}
+
+@media (min-width: 800px) {
+  .article {
+    margin: 3.5em auto;
+  }
+}
+
```
Comment (polosanya) [inline src/components/Article/Article.css:31]:
```suggestion
}
```
The problem is still present.
Leave only one newline in the end
image

**Review by polosanya — CHANGES_REQUESTED** [review]
Lint problem is not fixed. 
Feel free to ask for help in the chat if you have any difficulties

File: src/components/Article/Article.css (hunk 1 above)
Comment (dima-korovai) [reply src/components/Article/Article.css]: Hope now it's ok. At least run lint doesn't show up any error.

**Review by loralevitska — APPROVED** [review]
Good job!

**Pull request closed without merging**